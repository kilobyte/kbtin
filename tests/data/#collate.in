#collate x {2n n};#showme $x
#collate x {n s e};#showme $x
