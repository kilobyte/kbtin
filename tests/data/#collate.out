3n
n s e
